python/tribute.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+from paraview.simple import *
+
+# create a new 'PNG Series Reader'
+tributepng = PNGSeriesReader(FileNames=['tribute.png'])
+
+# create a new 'Calculator'
+calculator1 = Calculator(Input=tributepng)
+calculator1.ResultArrayName = 'originalData'
+calculator1.Function = 'sqrt(PNGImage_X*PNGImage_X+PNGImage_Y*PNGImage_Y)'
+
+# create a new 'Tetrahedralize'
+tetrahedralize1 = Tetrahedralize(Input=calculator1)
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=tetrahedralize1)
+tTKPersistenceDiagram1.ScalarField = ['POINTS', 'originalData']
+
+# create a new 'Threshold'
+threshold1 = Threshold(Input=tTKPersistenceDiagram1)
+threshold1.Scalars = ['CELLS', 'PairIdentifier']
+threshold1.LowerThreshold = -0.1
```

Comment:
I'd suggest to stick to the legacy format for the time being:
```
threshold1.ThesholdRange = [-0.1, 3560.0]
```

First, ParaView 5.10 is still a release candidate. 
Second, all TTK binary packages have been prepared against ParaView 5.9.

Once ParaView 5.10 is officially out and that we switch TTK's support to 5.10 officially too, we'll make a global pass on the examples to update the thresholds with a sed based script.

Reply:
Fixed